### PR TITLE
Fixing/Removing/Improving all Contribution Docs #2

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipulgupta2048@gmail.com>

**What this PR does / why we need it**:
More fixes in the docs of OpenEBS, find reference to #1511 

**Special notes for your reviewer**:
Please refer to #1511 

@mahebbar Here are the changes you requested.

### CODE REVIEW

**Comment by vipulgupta2048** [comment]
@kmova @mahebbar Should we merge this if no further changes are required?

File: contribute/design/README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ If you notice this classification, resembling a typical Container Orchestrator (
 
 OpenEBS Volumes provide persistent storage for containers, with resiliency against system failures, faster access to the storage, snapshot and backup capabilities. In addition, it provides a mechanism for monitoring the usage and enforcing QoS policies. 
 
-The Disks where data is persisted are called as *Storage Backends*, which can be either host directories, attached block devices or remote disks. Each OpenEBS Volume comprises of an iSCSI Target Container (represented as openebs-vol1 in the above diagram) and one or more Replica Containers (openebs-vol1-R1 and openebs-vol1-R2).
+The Disks where data is persisted are called as *Storage Backends*, which can be either host directories, attached block devices or remote disks. Each OpenEBS Volume comprises of an iSCSI Target Container (represented as openebs-vol1 in the above diagram) and one or more replicas Containers (openebs-vol1-R1 and openebs-vol1-R2).
```
Comment (kmova) [inline contribute/design/README.md]: Replica should be retained

**Comment by vipulgupta2048** [comment]
Done, all clean.

**Comment by NisantaSahoo** [comment]
Congratulations @vipulgupta2048  for your PR getting merged! Thanks for taking part in OpenEBS Hackfest.
Fill up the doc form to get the Hackfest T-Shirt and Swags
https://docs.google.com/spreadsheets/d/1TZYIdERnD72YPeUsdS5hDfI6xgXXCax4-DHpXh-o6r0/edit?usp=sharing
Looking for some more awesome contributions from you and win more prizes

**Pull request merged**